bin/dump.sh

```diff
@@ -3,7 +3,15 @@
 workdir=/tmp/work
 outputdir=/output
 re='^[0-9]+$'
+
 acme_file_path=${ACME_FILE_PATH:-/traefik/acme.json}
+certificate_file_name=${CERTIFICATE_FILE_NAME:-cert}
+certificate_file_ext=${CERTIFICATE_FILE_EXT:-.pem}
+certificate_file=${certificate_file_name}${certificate_file_ext}
+privatekey_file_name=${PRIVATEKEY_FILE_NAME:-key}
+privatekey_file_ext= ${PRIVATEKEY_FILE_EXT:-.pem}
+privatekey_file= ${privatekey_file_name}${privatekey_file_ext}
```

Comment:
```suggestion
privatekey_file_ext=${PRIVATEKEY_FILE_EXT:-.pem}
privatekey_file=${privatekey_file_name}${privatekey_file_ext}
```

We have to remove white spaces, as otherwise the variables are interpreted as commands.

Reply:
right, stupid mistake. although i am not a bash expert 😉